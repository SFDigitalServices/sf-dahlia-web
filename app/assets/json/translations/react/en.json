{
  "config.routePrefix": "",
  "errorBoundary.description": "An application error occurred. Check back later.",
  "errorBoundary.goHome": "Back to homepage",
  "footer.cityCountyOfSf": "City & County of San Francisco",
  "footer.contact": "Contact",
  "footer.dahliaDescription": "DAHLIA: San Francisco Housing Portal is a project of the <br /><a className='text-white' href='%{mohcdUrl}' target='_blank'>Mayor's Office of Housing and Community Development</a>",
  "footer.disclaimer": "Disclaimer",
  "footer.forGeneralQuestions": "For general program inquiries, you may call MOHCD at 415-701-5500.",
  "footer.forListingQuestions": "For listing and application questions, please contact the agent displayed on the listing.",
  "footer.giveFeedback": "Give Feedback",
  "footer.inPartnershipWith": "in partnership with<br /><a className='text-white' href='%{sfdsUrl}' target='_blank'>San Francisco Digital Services</a><br /><a className='text-white' href='%{mayorUrl}' target='_blank'>Mayor's Office of Civic Innovation</a>",
  "footer.privacyPolicy": "Privacy Policy",
  "idleTimeout.sessionExpired": "Your session has expired.",
  "idleTimeout.sessionInactivity": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information you've entered into the application for this listing if you choose not to respond.",
  "idleTimeout.sessionInactivityLoggedIn": "To protect your identity, your session will expire in one minute due to inactivity. You will lose any unsaved information and be logged out if you choose not to respond.",
  "idleTimeout.stayLoggedIn": "Stay logged in?",
  "label.clear": "Clear",
  "label.editEligbility": "Edit Eligibility",
  "languages.en": "English",
  "languages.es": "Español",
  "languages.tl": "Filipino",
  "languages.zh": "中文",
  "listingDetails.applicationDeadline.open": "Application Deadline %{date} at %{time}",
  "listingDetails.applicationDeadline.closed": "Applications Closed %{date} at %{time}",
  "listings.additional.hide": "Hide additional listings",
  "listings.additional.show": "Show additional listings",
  "listings.additional.subtitle": "We know you may have options about how many people will live with you. Here are listings for other household sizes and income levels.",
  "listings.additional.title": "Additional Listings",
  "listings.additionalInformation.header": "Additional Information",
  "listings.additionalInformation.subheader": "Required documents and selection criteria",
  "listings.applicationDeadline": "Application Deadline",
  "listings.applicationsClosed": "Applications Closed",
  "listings.atTotalIncome": "at <span class='eligibility-subtext'>$%{income}</span> %{per}",
  "listings.availableUnits": "Available Units",
  "listings.availableUnitsAndOpenWaitlist": "Available Units & Open Waitlist",
  "listings.buildingImageAltText": "A picture of the building",
  "listings.cc&r": "Covenants, Conditions and Restrictions (CC&R's)",
  "listings.cc&r_description": "The CC&R's explain the rules of the homeowners' association, and restrict how you can modify the property.",
  "listings.download_pdf": "Download PDF",
  "listings.eligibility.header": "Eligibility",
  "listings.eligibility.subheader": "Income, occupancy, preferences, and subsidies",
  "listings.eligibilityCalculator.matched": "Matched",
  "listings.eligibilityCalculator.notAMatch": "Not a Match",
  "listings.eligibilityCalculator.rent.noMatchingUnits": "Based on information you entered, you don't match any current listings for rent.",
  "listings.eligibilityCalculator.rent.showingMatchingUnits": "Showing matching units for rent",
  "listings.eligibilityCalculator.sale.noMatchingUnits": "Based on information you entered, you don't match any current listings for sale.",
  "listings.eligibilityCalculator.sale.showingMatchingUnits": "Showing matching units for sale",
  "listings.eligibilityCalculator.youMayBeEligible": "Based on information you entered, you may be eligible for units at the following property.",
  "listings.equal_housing_opportunity_logo": "Equal Housing Opportunity Logo",
  "listings.features.header": "Features",
  "listings.features.rentSubheader": "Amenities, unit details and additional fees",
  "listings.features.saleSubheader": "Amenities and unit details",
  "listings.forHouseholdSize": "for <span class='eligibility-subtext'>%{size}</span> %{people}",
  "listings.habitat.infoSession": "You will need to attend an information session.",
  "listings.habitat.payments": "Payments",
  "listings.habitat.payments.desc": "No down payment, but you do need to complete 500 hours of sweat equity. Your monthly payment will be 30% of your income.",
  "listings.important_program_rules": "Important Program Rules",
  "listings.includesPriorityUnits": "Includes Priority Units for %{priorities}",
  "listings.includingChildren": "(including <span class='eligibility-subtext'>%{number} %{children}</span> under 6)",
  "listings.lotteryResults.cardTitle": "Lottery Results Posted",
  "listings.lotteryResults.hide": "Hide Lottery Results",
  "listings.lotteryResults.noResults": "There are no listings with lottery results at this time.",
  "listings.lotteryResults.show": "Show Lottery Results",
  "listings.lotteryResults.subtitle": "You can now check lottery results on the remaining listings.",
  "listings.lotteryResults.title": "Lottery Results",
  "listings.monitored_by_mohcd": "Monitored by the Mayor's Office of Housing & Community Development",
  "listings.neighborhood.header": "Neighborhood",
  "listings.neighborhood.subheader": "Location and transportation",
  "listings.noMatches": "No Matches",
  "listings.noOpenListings": "No listings currently have open applications.",
  "listings.openWaitlist": "Open Waitlist",
  "listings.people": "people",
  "listings.person": "person",
  "listings.prioritiesDescriptor.hearing": "Hearing Impairments",
  "listings.prioritiesDescriptor.hearingVision": "Vision and/or Hearing Impairments",
  "listings.prioritiesDescriptor.mobility": "Mobility Impairments",
  "listings.prioritiesDescriptor.mobilityHearingVision": "Mobility, Hearing and/or Vision Impairments",
  "listings.prioritiesDescriptor.vision": "Vision Impairments",
  "listings.process.header": "Process",
  "listings.process.subheader": "Important dates and contact information",
  "listings.realtor_commission": "For the Buyer's Realtor",
  "listings.realtor_commission_header": "Your Commission: ",
  "listings.realtor_commission_how_to": "How to receive it: ",
  "listings.realtor_commission_not_eligible": "Your realtor is not eligible for a commission on this unit",
  "listings.re_pricing": "Resale Price Restrictions",
  "listings.required_documents": "Required Documents",
  "listings.reservedCommunityType.accessible": "Accessible Units Only",
  "listings.reservedCommunityType.artist": "Artist Loft Building",
  "listings.reservedCommunityType.disability": "Developmental Disability Building",
  "listings.reservedCommunityType.habitat": "Habitat Greater San Francisco",
  "listings.reservedCommunityType.senior": "Senior Building",
  "listings.reservedCommunityType.veteran": "Veterans Building",
  "listings.unitTypes.1 BR": "1 Bedroom",
  "listings.unitTypes.2 BR": "2 Bedroom",
  "listings.unitTypes.3 BR": "3 Bedroom",
  "listings.unitTypes.4 BR": "4 Bedroom",
  "listings.unitTypes.5 BR": "5 Bedroom",
  "listings.unitTypes.SRO": "SRO",
  "listings.unitTypes.Studio": "Studio",
  "listings.upcomingLotteries.hide": "Hide Upcoming Lotteries",
  "listings.upcomingLotteries.noResults": "There are no closed listings with upcoming lotteries at this time.",
  "listings.upcomingLotteries.show": "Show Upcoming Lotteries",
  "listings.upcomingLotteries.subtitle": "The application deadline has passed on the following listings, and you can still get information on the lottery date and time.",
  "listings.upcomingLotteries.title": "Upcoming Lotteries",
  "listings.special_notes": "Special Notes",
  "listings.waitlist.open": "Open Waitlist",
  "listings.weSuggestHousingCounselor": "For more help, we suggest talking with a housing counselor to explore your options. <a href='%{url}' target='_blank'>Find a housing counselor</a>.",
  "nav.accountSettings": "Account Settings",
  "nav.buy": "Buy",
  "nav.getAssistance": "Get Assistance",
  "nav.getFeedback": "We'd love to get <a target='_blank' href='%{feedbackUrl}'>your feedback.</a>",
  "nav.myAccount": "My Account",
  "nav.myApplications": "My Applications",
  "nav.myDashboard": "My Dashboard",
  "nav.myFavorites": "My Favorites",
  "nav.rent": "Rent",
  "nav.researchFeedback": "<a target='_blank' href='%{researchUrl}'>Sign up</a> to provide feedback on City websites",
  "nav.signIn": "Sign In",
  "nav.signOut": "Sign Out",
  "pageTitle.rentalListings": "Rental Listings",
  "pageTitle.saleListings": "Sale Listings",
  "pageTitle.signIn": "Sign In",
  "rentalDirectory.calloutbutton": "View opportunities",
  "rentalDirectory.callouttitle": "See more listings from our community partners.",
  "rentalDirectory.findMatchingListings": "Find matching listings",
  "rentalDirectory.ifYouTellUs": "If you tell us your household size and income, we can show you listings you may qualify for.",
  "rentalDirectory.orGetHelpCalculating": "or <a href='%{incomeLink}'>get help calculating your income</a>",
  "rentalDirectory.title": "Rent affordable housing",
  "saleDirectory.beforeApplying.makeSureYou": "Make sure you:",
  "saleDirectory.beforeApplying.readFullList": "Read <a href='%{url}' target='_blank'>our full list of requirements</a> for more details.",
  "saleDirectory.beforeApplying.step1": "Haven't owned residential property in the past 3 years",
  "saleDirectory.beforeApplying.step2": "Meet our <a href='%{url}' target='_blank'>income requirements</a>",
  "saleDirectory.beforeApplying.step3": "Complete <a href='%{url}' target='_blank'>homebuyer education</a>",
  "saleDirectory.beforeApplying.step4": "Get pre-approved for a mortgage loan by a <a href='%{url}' target='_blank'>lender on our list</a>",
  "saleDirectory.beforeApplying.step5": "Have enough in savings for closing costs and downpayment",
  "saleDirectory.beforeApplying.title": "Before applying",
  "saleDirectory.callout.citySecondLoan": "City Second Loan Program",
  "saleDirectory.callout.firstComeFirstServed": "First come, first served",
  "saleDirectory.callout.title": "Find more listings on our website:",
  "saleDirectory.getHelp.dalpProgramInfo": "The 2021 Downpayment Assistance Loan Program (DALP) will begin accepting applications on February 26, 2021.",
  "saleDirectory.getHelp.helpWithDownpayment": "Help with downpayments",
  "saleDirectory.getHelp.learnMoreAndHowToApply": "Learn more about DALP and how to apply on SF.gov",
  "saleDirectory.getHelp.mohcdPrograms": "The Mayor's Office of Housing and Community Development (MOHCD) offers a number of programs to help homebuyers in San Francisco.",
  "saleDirectory.getHelp.moreResources": "More resources",
  "saleDirectory.getHelp.otherGroupsMayQualify": "Others, like San Francisco teachers and first responders, may qualify for loans.",
  "saleDirectory.getHelp.somePeopleMayQualify": "Some people may qualify for downpayment assistance.",
  "saleDirectory.getHelp.title": "Get help",
  "saleDirectory.getHelp.viewFullListOfPrograms": "View the full list of programs",
  "saleDirectory.getHelp.visitMohcdPrograms": "Visit the MOHCD Affordable Homebuyer Programs page to learn more.",
  "saleDirectory.hoaDues": "HOA Dues",
  "saleDirectory.price": "Sales Price",
  "saleDirectory.seeTheListings": "See the listings",
  "saleDirectory.title": "Buy affordable housing",
  "signIn.badCredentials": "Invalid login credentials. Please try again.",
  "signIn.forgotPassword": "Forgot password?",
  "signIn.signedOutSuccessfully": "You have successfully logged out of your account.",
  "signIn.userTokenValidationTimeout": "We care about your security. We logged you out due to a connection issue. Please sign in to continue.",
  "t.available": "available",
  "t.child": "child",
  "t.children": "children",
  "t.close": "Close",
  "t.continue": "Continue",
  "t.dahliaSanFranciscoHousingPortal": "DAHLIA San Francisco Housing Portal",
  "t.homePage": "Home page",
  "t.income": "income",
  "t.incomeRange": "Income Range",
  "t.menu": "Menu",
  "t.minimumIncome": "Minimum Income",
  "t.numberRange": "%{minValue} to %{maxValue}",
  "t.perMonth": "per month",
  "t.perYear": "per year",
  "t.rent": "Rent",
  "t.seeDetails": "See Details",
  "t.skipToMainContent": "Skip to main content",
  "t.to": "to",
  "t.unitType": "Unit Type",
  "t.units": "Units",
  "t.waitlist": "Waitlist",
  "welcome.newListingEmailAlert": "Get emailed whenever a new listing is posted.",
  "welcome.seeRentalListings": "Rent",
  "welcome.seeSaleListings": "Buy",
  "welcome.signUpToday": "Sign Up today",
  "welcome.title": "Apply for affordable housing"
}
