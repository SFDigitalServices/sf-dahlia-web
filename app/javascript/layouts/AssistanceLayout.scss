.page-header {
    background-size: cover;
    background-position: center;
    padding: var(--bloom-s8);

    @media (min-width: $screen-md) {
        padding: var(--bloom-s12) 0;
    }

    .page-header__group {
        padding: 0 !important;
        width: full;

        @media (min-width: $screen-md) {
            text-align: start;
        }

        .page-header__title {
            font-size: var(--bloom-font-size-3xl);
            text-align: center !important;
            word-break: normal;
            padding-bottom: var(--bloom-s4);

            @media (min-width: $screen-md) {
                text-align: start !important;
                @apply w-2/3;
            }
        }

        .page-header__lead {
            text-align: center !important;
            word-break: normal;
            font-size: var(--bloom-font-size-sm);

            @media (min-width: $screen-md) {
                font-size: var(--bloom-font-size-lg);
                text-align: start !important;
                @apply w-2/3;
            }
        }
    }
}
