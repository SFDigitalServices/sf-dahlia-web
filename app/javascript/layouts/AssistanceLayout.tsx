import React from "react"
import Layout from "./Layout"
import { PageHeader, SidebarBlock, t } from "@bloom-housing/ui-components"
import { renderInlineMarkup, getSfGovUrl } from "../util/languageUtil"
import { ConfigContext } from "../lib/ConfigContext"

import "./AssistanceLayout.scss"

export interface Props {
  children: React.ReactNode
  title: string
  subtitle: string
  mainPage?: boolean
}

const AssistanceLayout = ({ children, title, subtitle, mainPage }: Props) => {
  const { getAssetPath } = React.useContext(ConfigContext)
  const classNames = mainPage
    ? "flex flex-wrap flex-col md:flex-row relative m-auto w-full"
    : "flex flex-wrap flex-col md:flex-row relative max-w-5xl m-auto w-full"
  return (
    <Layout title={title}>
      <PageHeader
        title={title}
        subtitle={subtitle}
        inverse
        backgroundImage={getAssetPath("bg@1200.jpg")}
      />
      <article className={classNames}>
        <div className="w-full md:w-2/3" data-testid="assistance-main-content">
          {children}
        </div>
        <SidebarBlock
          className="w-full md:w-1/3 md:max-w-xs text-gray-950 md:border-l border-t md:border-t-0 border-gray-450 border-b-0 p-6 mx-0"
          title={t("footer.contact")}
        >
          <span className="text-gray-950">
            <p className="mb-4">
              <span className="font-semibold">
                {t("assistance.contact.questionsAboutListings.title1")}
              </span>
              <br /> {renderInlineMarkup(t("assistance.contact.questionsAboutListings.subtitle"))}
            </p>
            <p className="mb-4">
              <span className="font-semibold">{t("assistance.contact.helpLine.title1")}</span>
              <br />
              {renderInlineMarkup(
                t("assistance.contact.helpLine.subtitle", {
                  url: getSfGovUrl("https://sf.gov/departments/311-customer-service-center", 217),
                })
              )}
            </p>
            <div>
              <span className="font-semibold">
                {t("assistance.contact.questionsAboutPrograms.title1")}
              </span>
              <br />
              <div>
                {renderInlineMarkup(
                  t("assistance.contact.questionsAboutPrograms.contactWebsite", {
                    url: getSfGovUrl(
                      "https://sf.gov/departments/mayors-office-housing-and-community-development",
                      55
                    ),
                  })
                )}
              </div>
              <div className="mt-1">
                {renderInlineMarkup(t("assistance.contact.questionsAboutPrograms.contactEmail"))}
              </div>
              <div className="mt-1">
                {t("assistance.contact.questionsAboutPrograms.contactCall.title")}
                <ul className="list-disc list-inside mt-1">
                  <li>
                    English: <a href="tel:+14157015622">(415) 701-5622</a>
                  </li>
                  <li>
                    Español: <a href="tel:+14157015624">(415) 701-5624</a>
                  </li>
                  <li>
                    中文: <a href="tel:+14157015623">(415) 701-5623</a>
                  </li>
                  <li>
                    Filipino: <a href="tel:+14157015570">(415) 701-5570</a>
                  </li>
                </ul>
              </div>
            </div>
          </span>
        </SidebarBlock>
      </article>
    </Layout>
  )
}

export default AssistanceLayout
