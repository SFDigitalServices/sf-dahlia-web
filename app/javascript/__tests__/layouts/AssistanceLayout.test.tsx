import React from "react"
import { within } from "@testing-library/react"
import AssistanceLayout, { languageToSFGovMap } from "../../layouts/AssistanceLayout"
import { renderAndLoadAsync } from "../__util__/renderUtils"
import renderer from "react-test-renderer"

const CHILD_CONTENT = "Content!"

describe("<AssistanceLayout />", () => {
  it("renders children", async () => {
    const { getByTestId } = await renderAndLoadAsync(
      <AssistanceLayout title="Title Text" subtitle="Subtitle Text">
        <h1>{CHILD_CONTENT}</h1>
      </AssistanceLayout>
    )
    const mainContent = getByTestId("assistance-main-content")

    expect(within(mainContent).getByText(CHILD_CONTENT)).not.toBeNull()
  })

  it("renders PageHeader", async () => {
    const TitleText = "Title Text"
    const SubtitleText = "SubTitle Text"
    const { getByText } = await renderAndLoadAsync(
      <AssistanceLayout title={TitleText} subtitle={SubtitleText}>
        <h1>{CHILD_CONTENT}</h1>
      </AssistanceLayout>
    )

    expect(getByText(TitleText)).not.toBeNull()
    expect(getByText(SubtitleText)).not.toBeNull()
  })

  it("lanauageToSFGovMap returns the proper url", () => {
    expect(languageToSFGovMap("en")).toBe(
      "https://sf.gov/departments/mayors-office-housing-and-community-development"
    )
    expect(languageToSFGovMap("es")).toBe("https://sf.gov/es/node/55")
    expect(languageToSFGovMap("zh")).toBe("https://sf.gov/zh-hant/node/55")
    expect(languageToSFGovMap("tl")).toBe("https://sf.gov/fil/node/55")
  })

  describe("Contact Bar", () => {
    it("renders Contact Information", () => {
      const tree = renderer
        .create(
          <AssistanceLayout title="Title Text" subtitle="Subtitle Text">
            <h1>{CHILD_CONTENT}</h1>
          </AssistanceLayout>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
