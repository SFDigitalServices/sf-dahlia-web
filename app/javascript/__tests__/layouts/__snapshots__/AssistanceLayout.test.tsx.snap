// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AssistanceLayout /> Contact Bar renders Contact Information 1`] = `
<div
  className="notranslate site-wrapper"
>
  <div
    className="site-content"
  >
    <title>
      Title Text
    </title>
    <meta
      charSet="utf-8"
    />
    <meta
      content="IE=edge"
      httpEquiv="X-UA-Compatible"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      content="M1zcm6GGM6sHSF_jvkq254DbYAj94JYbFC7ArZDAXlg"
      name="google-site-verification"
    />
    <meta
      content="Title Text"
      property="og:title"
    />
    <meta
      property="og:image"
    />
    <meta
      content="website"
      property="og:type"
    />
    <link
      rel="apple-touch-icon"
      sizes="57x57"
    />
    <link
      rel="apple-touch-icon"
      sizes="60x60"
    />
    <link
      rel="apple-touch-icon"
      sizes="72x72"
    />
    <link
      rel="apple-touch-icon"
      sizes="76x76"
    />
    <link
      rel="apple-touch-icon"
      sizes="114x114"
    />
    <link
      rel="apple-touch-icon"
      sizes="120x120"
    />
    <link
      rel="apple-touch-icon"
      sizes="144x144"
    />
    <link
      rel="apple-touch-icon"
      sizes="152x152"
    />
    <link
      rel="apple-touch-icon"
      sizes="180x180"
    />
    <link
      rel="icon"
      sizes="192x192"
      type="image/png"
    />
    <link
      rel="icon"
      sizes="32x32"
      type="image/png"
    />
    <link
      rel="icon"
      sizes="96x96"
      type="image/png"
    />
    <link
      rel="icon"
      sizes="16x16"
      type="image/png"
    />
    <link
      rel="manifest"
    />
    <meta
      name="msapplication-TileImage"
    />
    <header
      className="site-header"
    >
      <a
        className="site-header__skip-link"
        href="#main-content"
      >
        Skip to main content
      </a>
      <div
        className="language-bar"
      >
        <div
          className="language-bar__inner"
        >
          <nav
            aria-label="Choose your language"
            className="language-nav"
          >
            <ul
              className="language-nav__list"
            >
              <li>
                <button
                  className="language-nav__list-button is-active"
                  onClick={[Function]}
                  type="button"
                >
                  English
                </button>
              </li>
              <li>
                <button
                  className="language-nav__list-button"
                  onClick={[Function]}
                  type="button"
                >
                  Español
                </button>
              </li>
              <li>
                <button
                  className="language-nav__list-button"
                  onClick={[Function]}
                  type="button"
                >
                  中文
                </button>
              </li>
              <li>
                <button
                  className="language-nav__list-button"
                  onClick={[Function]}
                  type="button"
                >
                  Filipino
                </button>
              </li>
            </ul>
          </nav>
        </div>
      </div>
      <div
        className="site-header__notice "
      >
        <div
          className="site-header__notice-text"
        >
          <span>
            We'd love to get 
            <a
              href="https://airtable.com/shrw64DubWTQfRkdo"
              target="_blank"
            >
              your feedback.
            </a>
          </span>
        </div>
      </div>
      <nav
        aria-label="main navigation"
        className="site-header__container"
        role="navigation"
      >
        <div
          className="site-header__base "
        >
          <div
            className="site-header__logo-container site-header__logo-width-med"
          >
            <a
              aria-label="DAHLIA San Francisco Housing Portal"
              className="site-header__logo translate site-header__custom-width"
              href="/"
            >
              <div
                className="site-header__logo-content site-header__image-only-container"
              >
                <img
                  alt="Site logo"
                  className="site-header__logo-image site-header__image-only"
                />
              </div>
            </a>
          </div>
          <div
            className="site-header__navbar-menu"
          >
            <a
              className="site-header__link pb-4 pt-1 flex items-end "
              data-testid="Rent-0"
              href="/listings/for-rent"
            >
              <div>
                Rent
              </div>
            </a>
            <a
              className="site-header__link pb-4 pt-1 flex items-end "
              data-testid="Buy-1"
              href="/listings/for-sale"
            >
              <div>
                Buy
              </div>
            </a>
            <a
              className="site-header__link pb-4 pt-1 flex items-end "
              data-testid="My Favorites-2"
              href="/favorites"
            >
              <div>
                My Favorites
              </div>
            </a>
            <a
              className="site-header__link pb-4 pt-1 flex items-end "
              data-testid="Get Assistance-3"
              href="/get-assistance"
            >
              <div>
                Get Assistance
              </div>
            </a>
            <a
              className="site-header__link pb-4 pt-1 flex items-end "
              data-testid="Sign In-4"
              href="/sign-in"
            >
              <div>
                Sign In
              </div>
            </a>
          </div>
        </div>
      </nav>
    </header>
    <main
      className="md:overflow-x-hidden"
      data-testid="main-content-test-id"
      id="main-content"
    >
      <header
        className="page-header is-inverse"
      >
        <hgroup
          className="page-header__group"
        >
          <h1
            className="page-header__title"
            data-testid="page-header"
          >
            Title Text
          </h1>
          <p
            className="page-header__lead"
          >
             
            Subtitle Text
          </p>
        </hgroup>
      </header>
      <article
        className="flex flex-wrap flex-col md:flex-row relative max-w-5xl m-auto w-full"
      >
        <div
          className="w-full md:w-2/3"
          data-testid="assistance-main-content"
        >
          <h1>
            Content!
          </h1>
        </div>
        <section
          className="aside-block w-full md:w-1/3 text-gray-950 md:border-l border-t md:border-t-0 border-gray-450 p-6 mx-0"
        >
          <h4
            className="text__underline-weighted"
          >
            Contact
          </h4>
          <div
            className="text-sm text-gray-750"
          >
            <span
              className="text-gray-950"
            >
              <p
                className="mb-4"
              >
                <span
                  className="font-semibold"
                >
                  For questions about DAHLIA listings:
                </span>
                <br />
                 
                <span>
                  Contact the agent shown on the listing page.
                </span>
              </p>
              <p
                className="mb-4"
              >
                <span
                  className="font-semibold"
                >
                  For help with San Francisco services 24 hours a day, 7 days a week:
                </span>
                <br />
                 
                <span>
                  Call 
                  <a
                    href="tel:311"
                  >
                    311
                  </a>
                   or 
                  <a
                    href="https://sf311.org/"
                    target="_blank"
                  >
                    visit the 311 website.
                  </a>
                </span>
              </p>
              <div>
                <span
                  className="font-semibold"
                >
                  For questions about housing programs:
                </span>
                <br />
                <div>
                  <span>
                    <a
                      href="https://sf.gov/departments/mayors-office-housing-and-community-development"
                      target="_blank"
                    >
                      Visit the website for the Mayor’s Office of Housing and Community Development 
                    </a>
                    (MOHCD)
                  </span>
                </div>
                <div
                  className="mt-1"
                >
                  <span>
                    Email 
                    <a
                      href="mailto:sfhousinginfo@sfgov.org"
                    >
                      sfhousinginfo@sfgov.org
                    </a>
                  </span>
                </div>
                <div
                  className="mt-1"
                >
                  Call MOHCD at:
                  <ul
                    className="list-disc list-inside mt-1"
                  >
                    <li>
                      English: 
                      <a
                        href="tel:+14157015622"
                      >
                        (415) 701-5622
                      </a>
                    </li>
                    <li>
                      Español: 
                      <a
                        href="tel:+14157015623"
                      >
                        (415) 701-5623
                      </a>
                    </li>
                    <li>
                      中文: 
                      <a
                        href="tel:+14157015624"
                      >
                        (415) 701-5624
                      </a>
                    </li>
                    <li>
                      Filipino: 
                      <a
                        href="tel:+14157015570"
                      >
                        (415) 701-5570
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
            </span>
          </div>
        </section>
      </article>
    </main>
  </div>
  <footer
    className="site-footer"
  >
    <div
      className="footer-row "
    >
      <div
        className="footer-row__section pb-8"
      >
        <img
          alt=""
          data-testid="footer-logo-test-id"
        />
      </div>
    </div>
    <div
      className="footer-row "
    >
      <div
        className="footer-row__section py-2"
      >
        <p
          className="text-gray-500"
        >
          <span>
            DAHLIA: San Francisco Housing Portal is a project of the 
            <br />
            <a
              className="text-white"
              href="https://sf.gov/mohcd"
              target="_blank"
            >
              Mayor's Office of Housing and Community Development
            </a>
          </span>
        </p>
        <p
          className="text-xs mt-4 text-gray-500"
        >
          <span>
            in partnership with
            <br />
            <a
              className="text-white"
              href="https://digitalservices.sfgov.org/"
              target="_blank"
            >
              San Francisco Digital Services
            </a>
            <br />
            <a
              className="text-white"
              href="https://www.innovation.sfgov.org/"
              target="_blank"
            >
              Mayor's Office of Civic Innovation
            </a>
          </span>
        </p>
      </div>
    </div>
    <div
      className="footer-row "
    >
      <div
        className="footer-row__section pb-8"
      >
        <p
          className="text-sm"
        >
          For listing and application questions, please contact the agent displayed on the listing.
           
          <br />
          For general program inquiries, you may call MOHCD at 415-701-5500.
        </p>
      </div>
    </div>
    <section
      className="footer-sock"
    >
      <div
        className="footer-sock__inner"
      >
        <p
          className="footer-copyright"
        >
          © City & County of San Francisco
        </p>
        <nav
          aria-label="Footer"
          className="footer-nav"
        >
          <a
            className="text-gray-500"
            href="https://airtable.com/shrw64DubWTQfRkdo"
            target="_blank"
          >
            Give Feedback
          </a>
          <a
            className="text-gray-500"
            href="mailto:sfhousinginfo@sfgov.org"
          >
            Contact
          </a>
          <a
            className="text-gray-500"
            href="/disclaimer"
          >
            Disclaimer
          </a>
          <a
            className="text-gray-500"
            href="/privacy"
          >
            Privacy Policy
          </a>
        </nav>
      </div>
    </section>
  </footer>
</div>
`;
